BUILD

```diff
@@ -1,6 +1,13 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+# We use this to establish the build root, rather than `./pants`, because we cannot safely use the
+# latter as the sentinel filename per https://github.com/pantsbuild/pants/pull/8105.
+files(
```

Comment:
Rather than declaring this here, could we declare it somewhere else? `files` targets are included in the workspace relative... to a source root, I think? And if that's the case, you can declare this under a source root.

Reply:
> could we declare it somewhere else?

No, I don't think so. It must exist at the same directory level as `pants.ini`, because the location of `get_buildroot()` is used to determine where the buildroot is so that the options bootstrapper can then, for example, figure out other things like where `pants.ini` is located.